app/src/components/inputs/input/input.scss

```diff
@@ -44,3 +44,10 @@
     margin: 0;
   }
 }
+.disabled,
+.readonly {
```

Comment:
What the reason for style readOnly input like disabled? We can't use disabled state? It's allows read and copy text

Reply:
if we use disabled we can't use select() method